redaxo/src/addons/media_manager/pages/types.php

```diff
@@ -149,8 +149,8 @@
         $form = $ep->getParam('form');
         $sql = $form->getSql();
 
-        // remove delete button on internal types (status == 1)
-        if ($sql->getRows() > 0 && $sql->hasValue('status') && 1 == $sql->getValue('status')) {
+        // remove delete button on internal types
+        if ($sql->getRows() > 0 && $sql->hasValue('status') && rex_media_manager::STATUS_SYSTEM_TYPE == $sql->getValue('status')) {
```

Comment:
ich vermute das `hasValue` kann hier weg?

```suggestion
        if ($sql->getRows() > 0 && rex_media_manager::STATUS_SYSTEM_TYPE == $sql->getValue('status')) {
```

Reply:
Ja